### PR TITLE
Add res_pjsip_config_sangoma as external module under "res".

### PR DESCRIPTION
**Pull request by bkford**
Adds res_pjsip_config_sangoma as an option to menuselect for download.
It lives under the Resources tab in the external section.

### CODE REVIEW

**Comment by bkford** [comment]
cherry-pick-to: 21
cherry-pick-to: 20

File: tests/CI/buildAsterisk.sh [hunk 1]
```diff
@@ -145,6 +145,7 @@ if [ $NO_MENUSELECT -eq 0 ] ; then
 	runner menuselect/menuselect `gen_cats enable $cat_enables` menuselect.makeopts
 
 	mod_disables="res_digium_phone"
+	mod_disables="res_pjsip_config_sangoma"
```
Comment (gtjoseph) [inline tests/CI/buildAsterisk.sh:148]: This file is no longer used. Nothing in the CI directory is anymore.  It's on my list to remove.

You'll want to look at the asterisk-ci-actions repo in scripts.

File: res/res.xml [hunk 2]
```diff
@@ -12,3 +12,8 @@
 		</downloader>
 	</member_data>
 </member>
+<member name="res_pjsip_config_sangoma" displayname="Download the Sangoma PJSIP Configuration Module for Asterisk. See https://downloads.digium.com/pub/telephony/res_pjsip_config_sangoma.">
+	<support_level>external</support_level>
+	<conflict>no_binary_modules</conflict>
+	<defaultenabled>no</defaultenabled>
```
Comment (gtjoseph) [inline res/res.xml:18]: You need the depends for xmlstarlet and bash like res_digium_phone because the download script is dependent on them.

**Review by gtjoseph — CHANGES_REQUESTED** [review]

**Comment by gtjoseph** [comment]
This needs to be against master even though we don't download for it.  Then add 22 for cherry-picking.

**Comment by bkford** [comment]
Closing, switching to master

**Pull request closed without merging**